sdk/formrecognizer/azure-ai-formrecognizer/MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,244 @@
+# Guide for migrating azure-ai-formrecognizer to version 3.2.x from versions 3.1.x and below
+
+This guide is intended to assist in the migration to `azure-ai-formrecognizer (3.2.x)` from versions `3.1.x` and below. It will focus on side-by-side comparisons for similar operations between versions, please note that version `3.1.2` will be used for comparison with `3.2.0b1`.
+
+Familiariy with `azure-ai-formrecognizer (3.1.x and below)` package is assumed. For those new to the Azure Form Recognizer SDK please refer to the [README](README) rather than this guide.
+
+## Table of Contents
+- [Migration benefits](#migration-benefits)
+- [Important changes](#important-changes)
+- [Additional samples](#additional-samples)
+
+## Migration benefits
+
+A natural question to ask when considering whether or not to adopt a new version or library is what the benefits of doing so would be. As Azure Form Recognizer has matured and been embraced by a more diverse group of developers, we have been focused on learning the patterns and practices to best support developer productivity and add value to our customers.
+
+The benefit of the new design of the `azure-ai-formrecognizer (3.2.x)` library is that it introduces two new clients `DocumentAnalysisClient` and the `DocumentModelAdministrationClient` with unified methods for analyzing documents and provides support for the new features added by the service in version `2021-09-30-preview`.
+Please refer to the [README](README) for more information on these new clients. 
+
+## Important changes
+
+### Instantiating a client
+
+We continue to support connection strings and AAD authentication methods when creating our clients. Below are the differences between the two versions:
+
+- In `3.2.x`, we have added `DocumentAnalysisClient` and `DocumentModelAdministrationClient` which support service version `2021-09-30-preview`.
+- `FormRecognizerClient` and `FormTrainingClient` will return an error is called with an API version of `2021-09-30-preview` or above. 
+
+Creating a new client in `3.1.x`:
+```python
+form_recognizer_client = FormRecognizerClient(
+    endpoint=endpoint, credential=AzureKeyCredential(key)
+)
+```
+
+Creating a new client in `3.2.x`:
+```python
+document_analysis_client = DocumentAnalysisClient(
+    endpoint=endpoint, credential=AzureKeyCredential(key)
+)
+```
+
+### Analyzing documents
```

Comment:
it seems like there is too much here. I would prefer a specific section for prebuilt old vs new (receipt, businesscard, etc),
then another for recognizecontent.
the prebuilt-document is a new thing so it doesn;t have an equivalent before so no need to add it to this guide

Reply:
Reorganized a bit so that there isnt as much there. I'll think about separating into prebuilt and other content, adding an issue here: https://github.com/Azure/azure-sdk-for-python/issues/21001